### ISSUE TITLE
Can't add Ward of Radiance to Jacqueline's deck.

### ISSUE THREAD

**Issue by ericapfelsensei** [Kamalisk/arkhamdb#369]
Creating a new deck for Jacqueline, I noticed that Ward of Radiance isn't among the cards she can add.  I can't see any reason why she shouldn't have access to this level 0 mystic card.

**Comment by tdctaz** [comment]
I just checked, made a new deck with Jacqueline and added Ward of Radiance, it is both in the list and in the search.

Have you added `The Innsmouth Conspiracy` to your collection and deck list for your investigator?

**Comment by ericapfelsensei** [comment]
Yeah I finally realized after a week that I had to add Innsmouth to my
"owned" list for the cards to show up in the deckbuilder.
